app.js

```diff
@@ -40,58 +62,78 @@ storeSalesDataEl.appendChild(timeRowEl);
 var blankCell = document.createElement('td');
 timeRowEl.appendChild(blankCell);
 
-for (var cell = 0; cell < firstAndPike.businessHours.length; cell++) { // Needed time from first store to display hours.
-  var timeHour = document.createElement('td');
-  timeHour.textContent = firstAndPike.businessHours[cell];
-  timeRowEl.appendChild(timeHour);
+function creatingHeaderRow () {
+  for (var cell = 0; cell < firstAndPike.businessHours.length; cell++) { // Needed time from first store to display hours.
+    var timeHour = document.createElement('td');
+    timeHour.textContent = firstAndPike.businessHours[cell];
+    timeRowEl.appendChild(timeHour);
+  }
 }
 
-for (var storeIndex = 0; storeIndex < stores.length; storeIndex++) {
-  var tableRowEl = document.createElement('tr');
-  storeSalesDataEl.appendChild(tableRowEl);
-
-  var tableHeaderEl = document.createElement('th');
-  tableHeaderEl.textContent = stores[storeIndex].name;
-  tableRowEl.appendChild(tableHeaderEl);
-
-  for (var cookieData = 0; cookieData < stores[storeIndex].hourlyCookieCount.length; cookieData++){
-    var cookieCountEl = document.createElement('td');
-    cookieCountEl.textContent = stores[storeIndex].hourlyCookieCount[cookieData];
-    tableRowEl.appendChild(cookieCountEl);
+function creatingHeaderRow () {
+  for (var cell = 0; cell < firstAndPike.businessHours.length; cell++) { // Needed time from first store to display hours.
+    var timeHour = document.createElement('td');
+    timeHour.textContent = firstAndPike.businessHours[cell];
+    timeRowEl.appendChild(timeHour);
   }
+}
+creatingHeaderRow();
```

Comment:
Grouping function calls like this at the bottom will make it a lot easier to find where you are doing stuff.